tests/unit-tests/test_gnn.py

```diff
@@ -1525,6 +1536,7 @@ def create_lp_config(tmp_path, file_name):
             "gnn": {
                 "num_layers": 1,
                 "hidden_size": 4,
```

Comment:
For link prediction, can you test both num_layers = 1 and num_layers = 2?

Reply:
Add the test for num_layers=2 in test_link_prediction_weighted.